docs/HOWTO-bn.md

```diff
@@ -1,34 +1,36 @@
+* [ইউটিউব - মার্কডাউন ক্র্যাশ কোর্স](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
 
 
-কোন প্রশ্ন করতে দ্বিধাবোধ করবেন না। সব কন্ট্রিবিউটরই প্রথম PR থেকে শুরু করেছিল। তাহলে কেন না আমাদের [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community তে যোগদান করুন!
+কোন প্রশ্ন করতে দ্বিধাবোধ করবেন না; সমস্ত কন্ট্রিবিউটর তাদের প্রথম PR দিয়ে শুরু করেছিলেন। তাহলে, কেন আমাদের বড় এবং ক্রমবর্ধমান কমিউনিটি তে যোগদান করছেন না? [community](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) তে যোগদান করুন!
 
 <details align="center" markdown="1">
-<summary>Click to see the growth users vs. time graphs.</summary>
+<summary>সময়ের সাথে ব্যবহারকারীদের প্রবৃদ্ধির গ্রাফ দেখতে ক্লিক করুন।</summary>
 
 [![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
 
 [![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
 
 </details>
 
-এমনকি আপনি যদি একজন অভিজ্ঞ ওপেন-সোর্স কন্ট্রিবিউটর ও হয়ে থাকেন, কখনও কখনো কিছু জিনিস ভুল হতেই পারে। যখন আপনি আপনার PR সাবমিট করবেন ***GitHub Actions* আপনার কোড কে যাচাই-বাছাই করবে, কখনো বা স্পেসিং বা ক্যাপিটালাইজেশন এর মত ছোটখাটো জিনিস খুঁজে বের করবে**। যদি আপনি সবুজ বাটন পেয়ে যান, তাহলে বুঝতে পারবেন সবকিছু রিভিউ এর জন্য প্রস্তুত। কিন্তু যদি আপনি সবুজ বাটন না পান তাহলে ফেইল্ড হওয়া চেক এর নিচে "Details" এ ক্লিক করলে সমস্যাগুলি খুঁজে বের করতে পারবেন। তারপর সেই সমস্যাগুলো ফিক্স করার পর আপনার PR এ কমিট করবেন।
+এমনকি আপনি যদি একজন অভিজ্ঞ ওপেন-সোর্স কন্ট্রিবিউটর ও হয়ে থাকেন, কখনও কখনো কিছু জিনিস ভুল হতেই পারে। যখন আপনি আপনার PR সাবমিট করবেন GitHub Actions আপনার কোড কে যাচাই-বাছাই করবে, কখনো বা স্পেসিং বা ক্যাপিটালাইজেশন এর মত ছোটখাটো জিনিস খুঁজে বের করবে। যদি আপনি সবুজ বাটন পেয়ে যান, তাহলে বুঝতে পারবেন সবকিছু রিভিউ এর জন্য প্রস্তুত। কিন্তু যদি আপনি সবুজ বাটন না পান তাহলে ফেইল্ড হওয়া চেক এর নিচে "Details" এ ক্লিক করলে সমস্যাগুলি খুঁজে বের করতে পারবেন। তারপর সেই সমস্যাগুলো ফিক্স করার পর আপনার PR এ কমিট করবেন।
 
-যদি আপনার সন্দেহ হয় যে আপনার রিসোর্স `Free-Programming-Books` এর জন্য উপযুক্ত কিনা, এই গাইডলাইন্স পড়ে দেখুন- [CONTRIBUTING](CONTRIBUTING.md) ([translations](README.md#translations))
+শেষমেশ, আপনি যদি নিশ্চিত না হন যে আপনি যে রিসোর্সটি যোগ করতে চান তা Free-Programming-Books এর জন্য উপযুক্ত কিনা, তাহলে কোনও সন্দেহ থাকলে CONTRIBUTING এ উল্লেখিত নির্দেশনাগুলি পড়ে দেখুন।
```

Comment:
পরিশেষে, আপনি যে রিসোর্সটি যোগ করতে চান তা Free-Programming-Books এর জন্য উপযুক্ত কিনা সেই বিষয়ে নিশ্চিত না হলে, [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) এর উল্লেখিত নির্দেশনাগুলি পড়ে দেখুন।

Reply:
Thanks for the suggestion but my request already have merged with a different branch